siddhi-extensions/geo/geo-event-fuser/src/main/java/org/wso2/cep/geo/libs/ExecutionPlansCount.java

```diff
@@ -0,0 +1,41 @@
+package org.wso2.cep.geo.libs;
+
+/*
+*  Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Thanks @lahirue , This code has been refactored and no longer available in the repository, Since could not fix the issue.